### PR TITLE
[DQM] Various fixes/improvements for unit test

### PR DESCRIPTION
**Pull request by smuzaffar**
- Convert unit tests to use `<test ...command="cmd"/>` instead of using `FW unit tests driver`
- Make sure that test can run from its dedicated test directory.

### CODE REVIEW

**Comment by emanueleusai** [comment]
+1

File: Validation/CTPPS/test/BuildFile.xml [hunk 1]
```diff
@@ -1,4 +1 @@
-<bin name="TestCTPPSDirectProtonSimulation" file="TestDriver.cc">
```
[the remark this answers is not in the document]
Comment (smuzaffar) [reply Validation/CTPPS/test/BuildFile.xml:1]: ah right @perrotta, yes it should be removed. I can take care of it in a separate PR. I will be doing a final round of updates for unit tests

**Comment by perrotta** [comment]
+1
- The now unused Validation/CTPPS/test/TestDriver.cc will get removed in a separate PR

File: Validation/CTPPS/test/BuildFile.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (smuzaffar) [reply Validation/CTPPS/test/BuildFile.xml:1]: this has been removed in https://github.com/cms-sw/cmssw/pull/41152

**Pull request merged**